src/components/date-range/date-range.css

```diff
@@ -0,0 +1,4 @@
+.date-input:hover {
+  border-color: black;
+  border-bottom-width: 2px;
+}
```

Comment:
this could be replaced by `tailwindcss` with: `'hover:border-back hover:border-b-2'`

Reply:
doesn't work with border width :( I added hover for the borderWidth attribute and still doesn't work. So I decided to have both together although I can move border-color color out that one works

Reply:
🤔  did you add to the `variants` section?
i added to the `variants` section this: `borderWidth: ['responsive', 'hover']` and it seems to be working on my side

Reply:
yea I did let me try again maybe I forgot to save lol

Reply:
oki lol, and don't forget to run `yarn build-tailwindcss` to re-build the css after you saved the config :)

Reply:
that was the setup I was missing hehehe thanks~! it works now